ios/wrappers/utils/CioConfigUtils.swift

```diff
@@ -56,3 +56,58 @@ func cioInitializeConfig(from config: RCTCioConfig, logLevel: String?) -> CioCon
     
     return CioConfig(cdp: cdpConfigBuilder.build(), inApp: inAppConfig)
 }
+
+// To convert dictionary to Codable
+struct AnyCodable: Codable {
+    var value: Any
+    
+    init(_ value: Any) {
+        self.value = value
+    }
+    
+    init(from decoder: Decoder) throws {
+        let container = try decoder.singleValueContainer()
+        
+        if let value = try? container.decode(String.self) {
+            self.value = value
+        } else if let value = try? container.decode(Int.self) {
+            self.value = value
+        } else if let value = try? container.decode(Double.self) {
+            self.value = value
+        } else if let value = try? container.decode(Bool.self) {
+            self.value = value
+        } else if let value = try? container.decode([String: AnyCodable].self) {
```

Comment:
i think it should be handling `[String: Any]` here too, no? its for the case the where a map is sent in value

Reply:
Great suggestion. Yes it should also cater `[String: Any]` to handle nested dictionary. I realised that we might not need `[String: AnyCodable]` as we won't receive it from the JS/TS user so removed it.